edsdme/scripts/utils.js

```diff
@@ -359,6 +376,12 @@ function getComplexQueryParams(el, collectionTag) {
 
   const collectionTagsStr = collectionTags.filter((e) => e.length).join('+AND+');
   let resulStr = `(${collectionTagsStr})`;
+
+  const qaContentTag = '"caas:adobe-partners/qa-content"';
+  if (!checkForQaContent(el)) {
```

Comment:
Wouldn't be easier to check for the whole tag in the result string at the end `resulStr.includes('""caas:adobe-partners/qa-content"')` and if this is false just to add `+NOT+...` ?